testing/trino-server-dev/etc/event-listener.properties

```diff
@@ -1,3 +0,0 @@
-event-listener.name=http
```

Comment:
Do we lose test coverage?

Reply:
No, because, sadly, `testing/trino-server-dev/etc/` is not read on CI. The dev server may or may not work.

Reply:
The development configs are only for running locally in the IDE.

Reply:
> The development configs are only for running locally in the IDE.

they are for running locally in the IDE, but they should not be used *only* there
otherwise they tend to break without people being aware of that

but yes, test coverage must not depend on them